unstable/zwlr-screencontent-v1.xml

```diff
@@ -0,0 +1,209 @@
+      <entry name="byte_buffer" value="0"
+             summary="file descriptors hold mappable linear memfd buffers"/>
+      <entry name="dma_buf" value="1"
+             summary="file descriptors contain valid dmabuf objects"/>
+    </enum>
+
+    <enum name="frame_flags">
+      <description summary="frame flags">
+        Special flags that must be respected by the client.
+        Transient frames indicate short lifetime frames (such as swapchain
+        images from external clients). Clients are advised to copy them and do
+        all processing outside of the "ready" event.
+      </description>
+      <entry name="transient" value="1"/>
+    </enum>
+
+    <event name="receive">
+      <description summary="main callback">
+        Main callback supplying the client with information about the frame,
+        as well as an object to serve as context for destruction. Always called
+        first before any other events.
+      </description>
+      <arg name="width" type="uint"
+           summary="frame width, scaling factor included"/>
+      <arg name="height" type="uint"
+           summary="frame height, scaling factor included"/>
+      <arg name="scale" type="uint"
+           summary="current scaling factor of the client"/>
+      <arg name="type" type="uint" enum="frame_type"
+           summary="indicates the type of frame preferred"/>
+      <arg name="frame_flags" type="uint" enum="frame_flags"
+           summary="indicates special frame features"/>
+      <arg name="mod_high" type="uint"
+           summary="drm format modifier, high"/>
+      <arg name="mod_low" type="uint"
+           summary="drm format modifier, low"/>
+      <arg name="num_objects" type="uint"
+           summary="indicates how many objects (FDs) the frame has (max 4)"/>
+      <arg name="num_layers" type="uint"
+           summary="indicates how many layers the frame has (max 4)"/>
+    </event>
+    <event name="receive_object">
+      <description summary="object receiving callback">
+        Callback which serves to supply the client with the file descriptors
+        containing the data for each object.
+        The amount of data in each FD can be inferred via lseek().
+        Will be called after the main receive event, once per object.
+      </description>
+      <arg name="index" type="uint"
+           summary="index of the current object"/>
+      <arg name="fd" type="fd"
+           summary="fd of the current object"/>
+    </event>
+    <event name="receive_layer">
+      <description summary="layer receiving callback">
+        Callback which serves to supply the client with information on what's
+        contained in each file descriptor and how its laid out.
+        Will be called after the main receive event, once per layer.
+      </description>
+      <arg name="index" type="uint"
+           summary="index of the current plane"/>
+      <arg name="format" type="uint"
+           summary="format of the layer (DRM_FORMAT_*)"/>
```

Comment:
There's an enum for this

Reply:
This one: https://wayland.freedesktop.org/docs/html/apa.html#protocol-spec-wl_shm-enum-format

Yeah, it is designed to exactly match `DRM_FORMAT_*`.